packages/devtools-launchpad/src/index.js

```diff
@@ -107,30 +133,26 @@ function getTargetFromQuery() {
 async function bootstrap(React, ReactDOM, App, appActions, appStore) {
   const connTarget = getTargetFromQuery();
   if (connTarget) {
-    return startDebugging(connTarget, appActions)
-      .then(({ tab, client }) => {
-        debugGlobal("client", client.clientCommands);
-        renderRoot(React, ReactDOM, App, appStore);
-        return { tab, connTarget, client };
-      });
+    const { tab, client } = await startDebugging(
+      connTarget, appActions
+    );
+
+    await updateConfig();
+    debugGlobal("client", client.clientCommands);
+    renderRoot(React, ReactDOM, App, appStore);
+    return { tab, connTarget, client };
   }
 
   const { store, actions, LaunchpadApp } = await initApp();
   renderRoot(React, ReactDOM, LaunchpadApp, store);
-  chrome.connectClient().then(tabs => {
-    actions.newTabs(tabs);
-  }).catch(e => {
-    console.log("Connect to chrome:");
-    console.log("https://github.com/devtools-html/debugger.html/blob/master/CONTRIBUTING.md#chrome");
-  });
+  const chromeTabs = await chrome.connectClient();
+  actions.newTabs(chromeTabs);
 
-  chrome.connectNodeClient().then(tabs => {
-    actions.newTabs(tabs);
-  });
+  const nodeTabs = await chrome.connectNodeClient();
+  actions.newTabs(nodeTabs);
 
-  return firefox.connectClient().then(tabs => {
-    actions.newTabs(tabs);
-  });
+  const firefoxTabs = await firefox.connectClient();
```

Comment:
Nice catch. this should go back to the way it was where they would run concurrently